ssl/s3_cbc.c

```diff
@@ -196,6 +198,9 @@ int ssl3_cbc_digest_record(const EVP_MD *md,
         return 0;
 
     if (EVP_MD_is_a(md, "MD5")) {
+#ifdef FIPS_MODULE
+        return 0;
+#else
         if (MD5_Init((MD5_CTX *)md_state.c) <= 0)
             return 0;
         md_final_raw = tls1_md5_final_raw;
```

Comment:
Why doesn't this generate a defined but not used error when building FIPS?

Both `u32toLE` and `tls1_md5_final_raw()` ought to be conditioned on FIPS.

Reply:
tls1_md5_final_raw() already was.. u32toLE is a macro.. I have moved it inside the #ifndef